kolibri/core/assets/src/state/store.js

```diff
@@ -108,6 +108,10 @@ export const coreMutations = {
   SET_LOGGING_COMPLETION_TIME(state, time) {
     state.core.logging.summary.completion_timestamp = time;
   },
+  SET_LOGGING_CONTENT_STATE(state, contentState) {
+    // TODO: Consider whether we want to save these to the session log as well.
```

Comment:
Probably not, as we would never reload state from this, so this would be entirely for analytics purposes.